### PR TITLE
ba-143 fix bug with recipe add edit buttons

### PR DESCRIPTION
**Pull request by patrick-hebner**

### CODE REVIEW

**Comment by flashback2k14** [comment]
lgtm

File: app/elements/socobo-recipe/socobo-recipe-details-add.html [hunk 1]
```diff
@@ -31,56 +31,38 @@
 
       /* CONTENT */
       .wrapper {
-        height: 75%;
-        max-height: 85%;
         width: 90%;
         min-width:700px;
         padding: 0;
-        overflow: auto;
+        /*overflow: auto;*/
```
Comment (lyio) [inline app/elements/socobo-recipe/socobo-recipe-details-add.html]: Can you remove this line if it is no longer needed?

File: app/elements/socobo-recipe/socobo-recipe-details-add.html [hunk 2]
```diff
@@ -203,101 +184,99 @@
...
-            label="Here you might place a brief description or the like ..."
-            value="{{recipe.text}}">
-          </paper-textarea>
-          <!-- DETAIL STEPS -->
-          <div class="subHeadings">Steps</div>
-          <hr class="separator"/>
-          <div class="steps" id="steps_input">
-            <template is="dom-repeat" items="{{recipe_inputs}}">
-              <socobo-recipe-details-step
+      <paper-dialog-scrollable class="scrollableContent">
```
Comment (lyio) [inline app/elements/socobo-recipe/socobo-recipe-details-add.html:187]: I guess this is basically indentation?

Comment (patrick-hebner) [inline app/elements/socobo-recipe/socobo-recipe-details-add.html:187]: I moved the content which should be scrollable into the paper-dialog-scrollabe after I had built it in. The purpose of this was to create a button area which is stuck to the bottom and doesn't overlay other gui elements

**Comment by flashback2k14** [comment]
lgtm

**Comment by lyio** [comment]
lgtm

**Pull request merged**